asreview/webapp/app.py

```diff
@@ -165,19 +165,15 @@ def api_boot():
         """Get the boot info."""
 
         authenticated = not app.config.get("LOGIN_DISABLED", False)
+        login_info = app.config.get("LOGIN_INFO", False)
 
         response = {
             "authentication": authenticated,
             "version": asreview_version,
+            "login_info": login_info,
         }
 
         if authenticated:
-            # if recaptcha config is provided for account creation
-            if app.config.get("RE_CAPTCHA_V3", False):
-                response["recaptchav3_key"] = app.config["RE_CAPTCHA_V3"].get(
-                    "KEY", False
-                )
-
```

Comment:
Can you explain?

Reply:
Nothing is done with it. I have started it, but nothing concrete is done with it in the frontend

Reply:
Oke clear.